src/EcommerceStatsBundle/Resources/config/services.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="true" />
```

Comment:
Shouldn't be needed.

Only the (to be added) controller needs to be public.

Reply:
Also below.

Reply:
Good catch, I missed that. 

:heavy_check_mark: Removed the `public` definitions.